packages/strapi-plugin-upload/admin/src/containers/InputModalStepperProvider/index.js

```diff
@@ -296,7 +305,7 @@ const InputModalStepperProvider = ({
 
   const getFilters = () => {
     const compactedParams = compactParams(params);
-    const searchParams = generateSearchFromFilters(compactedParams, ['_limit', '_sort', '_start']);
+    const searchParams = generateSearchFromFilters(compactedParams);
```

Comment:
You should add the filters to remove to the `getFilters` and pass the option in the `fetchMediaLibFilesCount`

```
const fetchMediaLibFilesCount = () => {
  const paramsToSend = getFilters(['_limit', '_sort', '_start']);
}

const getFilters = (filtersToOmit = []) => {
  // ...
  const searchParams = generateSearchFromFilters(compactedParams, filtersToOmit);
}
```